config_LST1_muons.cfg

```diff
@@ -100,53 +79,38 @@ ARRANG -4.84             // La Palma, 01/12/2021
 *                         (re-centered on new central MST (MST 15) position).
 *
 * LSTs:
-*
-* #define LSTHT  16.00E2
-* #define LSTRAD 12.50E2
-* 
-* TELESCOPE    0.0E2    0.0E2  $(LSTHT)    $(LSTRAD)  # LST
-TELESCOPE    0.0E2    0.0E2  16.00E2    12.50E2  // LST
+TELESCOPE 0.0E2    0.0E2  16.00E2    12.50E2  // LST
 CSCAT  1  9.8E2  0.     # To 80% of dish radius
```

Comment:
Should we make the scatter radius here configurable? There is surely a template option for this, right?

Reply:
Indeed there is. We can make NSCAT configurable as well